patches/tModLoader/Terraria/Main.cs.patch

```diff
@@ -760,16 +760,30 @@
  
  		public T OurLoad<T>(string path) {
```

Comment:
Can you reduce patch size by making a wrapper method, and just renaming this to `OurLoad_`?